frontend/app/assets/javascripts/spree/frontend/views/spree/products/index.js

```diff
@@ -17,16 +17,35 @@ Spree.ready(function($) {
     $(this).toggleClass('plp-overlay-card-item--selected')
   })
 
-  $('.color-select').click(function() {
-    var allOptionsBorders = $('.color-select-border')
-    var borderElement = $(this).find('.color-select-border')
+  var allOptionsBorders = $('.color-select-border')
+  var addToCartButton = document.getElementById('add-to-cart-button')
```

Comment:
Let's use jQuery instead of vanillaJS. We should avoid mixing content.
`var addToCartButton = $('#add-to-cart-button')`